git-lfs/git-lfs.install

```diff
@@ -0,0 +1,21 @@
+post_install() {
+  for arch in mingw32 mingw64; do
+    if [ -f "${arch}/bin/git-lfs.exe" ]; then
+      git config --system filter.lfs.clean "git-lfs clean %f"
+      git config --system filter.lfs.smudge "git-lfs smudge %f"
+      git config --system filter.lfs.required true

```

Comment:
@technoweenie thanks - and I missed the fact v1.2 has just shipped so I'll make sure we're including that!